src/com/intellij/rt/coverage/data/LineData.java

```diff
@@ -41,7 +41,6 @@ public LineData(final int line, final String desc) {
 
   public void touch() {
     myHits++;
-    setTestName(ProjectData.getCurrentTestName());
```

Comment:
won't old instrumenter break?

Reply:
No, as I set test name in com.intellij.rt.coverage.data.ProjectData#testEnded.

Reply:
Moving test name set to the dumping stage reduces time from 10s to 7.3 in new tracing for joda.time. So, I'd make this test name set at the dumping stage. Also, I'll try to make trace saving on a separate thread.

Reply:
Parallel saving does not lead to a significant time decrease, so it was not implemented for now